Library/Formula/advancecomp.rb

```diff
@@ -1,5 +1,5 @@
 class Advancecomp < Formula
-  desc "Recompression utilities for .PNG, .MNG, .ZIP, and .GZ files"
+  desc "Recompression utilities for .PNG, .MNG, .ZIP, and .GZ files (advzip, advpng, advmng and advdef)"
```

Comment:
The `desc` field is limited to 80 characters; this is a bit too long. As an alternative, maybe creating aliases under those names would make this more discoverable?